lib/pages/main/tab_settings/tab_settings.dart

```diff
@@ -0,0 +1,142 @@
+
+  String settingsUnlockLabel = "";
+  Widget settingsUnlockIcon = const Icon(Icons.fingerprint);
+  bool isFirstOpen = true;
+
+  @override
+  void didChangeDependencies() {
+    if (isFirstOpen) {
+      biometricService.getAvailableBiometric(context).then((result) {
+        setState(() {
+          settingsUnlockLabel = result['label'];
+          settingsUnlockIcon = result['icon'];
+        });
+      });
+      isFirstOpen = false;
+    }
+    super.didChangeDependencies();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final l10n = l10nOf(context);
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(l10n.appTabSettings, style: TextStyles.textExtraLargeBold),
+        backgroundColor: Colors.transparent,
+      ),
+      body: Padding(
+        padding: ThemeEdgeInsets.pageInsets,
+        child: Column(
+          children: [
+            Expanded(
+              child: SingleChildScrollView(
+                child: Column(
+                  children: [
+                    SettingsListTile(
+                      prefix: const Icon(
+                        Icons.lock,
+                        color: LightThemeColors.textColorSecondary,
+                      ),
+                      title: l10n.settingsLockWallet,
+                      suffix: const SizedBox.shrink(),
+                      onPressed: () {
+                        appStore.reportGlobalError("");
+                        appStore.reportGlobalNotification("");
+                        appStore.setCurrentTabIndex(
+                            0); // so after unlock, it goes to Home
+                        appStore.signOut();
+                        appStore.checkWalletIsInitialized();
+                        timedController.stopFetchTimers();
+                        context.go('/signInNoAuth');
+                      },
+                    ),
+                    SettingsListTile(
+                      prefix: SvgPicture.asset(AppIcons.autoLock, height: 20),
+                      title: l10n.settingsLabelAutlock,
+                      path: AutoLockSettings(),
+                    ),
+                    SettingsListTile(
+                      prefix: SvgPicture.asset(AppIcons.export, height: 20),
+                      title: l10n.settingsLabelExportWalletVaultFile,
+                      path: const ExportWalletVault(),
+                    ),
+                    SettingsListTile(
+                      prefix:
+                          SvgPicture.asset(AppIcons.changePassword, height: 20),
+                      title: l10n.settingsLabelChangePassword,
+                      path: const ChangePassword(),
+                    ),
+                    SettingsListTile(
+                      prefix: settingsUnlockIcon,
+                      title: settingsUnlockLabel,
+                      path: const ManageBiometrics(),
+                    ),
+                    SettingsListTile(
```

Comment:
Wallet connect can't be included yet because we don't have the functionality in the develop.
Please remove (or comment) all references to this.